vector/src/main/java/im/vector/app/features/home/room/detail/composer/RichTextComposerLayout.kt

```diff
@@ -191,8 +197,7 @@ class RichTextComposerLayout @JvmOverloads constructor(
         }
 
         views.composerFullScreenButton.apply {
-            // There's no point in having full screen in landscape since there's almost no vertical space
-            isInvisible = resources.configuration.orientation == Configuration.ORIENTATION_LANDSCAPE
```

Comment:
What about tablets here? Maybe add a boolean in the resource to take care of that?